### ISSUE TITLE
Update to newer versions?

### ISSUE THREAD

**Issue by Wordsmith9091** [RenoFischa/idrive#4]
Hi! This docker image has been super-helpful to me, as it let me set up a standard Linux install of idrive on my Qnap system, instead of needing to use the Qnap GUI version (which hasn't worked as well for me).

It appears this is now several minor release versions behind the official idrive releases, though. Any chance of updating to 3.7 (the current release)?

Or perhaps there's a way to tweak the package to let idrive's own updater function within the container?

**Comment by RenoFischa** [comment]
I've pushed an updated image to Dockerhub. Relogin might be required.

**Comment by heypoojman** [comment]
I agree with Wordsmith9091, your docker image has been very helpfull and thanks for building it.  I pulled the latest docker image but it still pulls the 3.2 version.  I used the 3.7 tag instead of using latest and it installs 3.7.  On the command line, the new version is running (i.e., dashboard, cron, cdp running) and seems to work from the command line but the web interface stops working (showing the computer offline).  If I go back to the version 3.2, it shows online again.  I have also tried a new docker container with a fresh install of 3.7.  Not sure if you have any suggestions.
![image](https://github.com/user-attachments/assets/ab22c763-cb0b-4ba5-9ac7-d65a3d99a5d2)

Thanks

**Comment by Wordsmith9091** [comment]
Thanks so much! But I'm running into the same problem as heypoojman on 3.7, in my case on a Qnap system (but with the container administered via Portainer, not Qnap's Container Station). Even after logging out and logging in on the command line, the idrive web interface doesn't seem to recognize the system as connected or reflect any activity. On the command line in the container, it appears logged in and can get information from the account. I reverted to 3.2 and the web interface once again shows the system online.

(Note, though: I was able to move to 3.7 with "latest," unlike heypoojman)

**Comment by RenoFischa** [comment]
@heypoojman You mentioned that you've tried with a new docker container with a fresh install of 3.7. Does the web interface show online or offline there? If online, which version?
My web interface still says 2.38 and offline, but my scheduled work just fine, so I've never looked into that yet.

**Comment by heypoojman** [comment]
The web interface shows offline with 3.7.  However, I was able to re-run the yaml file with the latest tag, 3.2 was installed and everything showed online in the web interface.  I was able to schedule backups and everything seems to work via command line.

**Comment by Wordsmith9091** [comment]
My experience is slightly different than heypoojman.

"Latest" did indeed update my install to 3.7, according to the command line. I was able to do operations at the command line, including initiating a backup (I didn't have any changed files, so none were updated, but it didn't error out or anything). However, the web interface continued to show 3.2 and offline. 

I had to explicitly use the 3.2 tag to downgrade back to 3.2, after which the web interface immediately recognized the computer as online (and still at 3.2)

**Comment by heypoojman** [comment]
I just stopped/started the docker container using the latest tag in my YAML file.  It is running 3.7 and indicates online in the web interface now but it still shows version 3.2.0 in the web interface:

![image](https://github.com/user-attachments/assets/0c3527ff-0650-4633-bd90-badca0d4ad4b)

![image](https://github.com/user-attachments/assets/dc459707-463c-43ab-adaa-1414316b3218)

**Comment by fitch22** [comment]
Just another data point.  I upgraded to 3.7 by pulling the latest image.  I am running via docker on Truenas Scale 24.10.0.2.

My backups seem to be running OK, I get an email with status from each.  But I just went to the idrive web site to see, and my computer shows version 3.2 and offline.  What is also strange, is that there are no logs from 3.7 available on the web, the latest log file shown is from when I was running 3.2.

**Comment by Wordsmith9091** [comment]
I wound up migrating my setup to a similar container that's here: https://github.com/snorre-k/idrive-docker

Because the basic setup was the same as yours, it didn't take much to do the change -- I just had to account for the differences in volumes in the Docker Compose but they depend on the same locations within the container. This one does indeed seem to be showing me as 3.7 in the web interface, and online.

I'm not sure what's different in that version than this one, but giving it a look might give a clue to whatever's not quite working as expected.

**Comment by bcurran3** [comment]
There is a 3.8 release now. :-)

![Image](https://github.com/user-attachments/assets/ed603aa0-e3b0-4061-986c-74d79aee49e4)

**Comment by heypoojman** [comment]
Try using the "snorre0815/idrive-docker:latest" docker container.  It has been staying with the latest iDrive versions.

Jerry
________________________________
